src/Modules/Reports/Commands.Reports/MicrosoftPowerBIMgmt.Reports.psd1

```diff
@@ -71,7 +71,7 @@ PowerShellVersion = '3.0'
 FunctionsToExport = '*'
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'Get-PowerBIReport'
+CmdletsToExport = 'Get-PowerBIReport', 'Export-PowerBIReport', 'Get-PowerBIDashboard', 'Get-PowerBITile', 'Get-PowerBIImport'
```

Comment:
We don't necessarily need to have the modules mirror the structure defined by the SDK, but given that our API documentation is based off the SDK now, this might be the most customer-friendly thing to do.
See: https://docs.microsoft.com/en-us/rest/api/power-bi/

Get-PowerBIDashboard and Get-PowerBITitle might be best suited to live in a Dashboards module. Get-PowerBIImport might be best suited to live in a Imports Module. I realize this is likely more setup work, but I think it's important we get this right first.

It also lines up with your roadmap: https://github.com/Microsoft/powerbi-powershell/blob/master/ROADMAP.md

Reply:
OK, I follow your reasoning here. I believe Dashboards are actually the highest parent level object in Power BI (Dashboards have Tiles which have Reports which have Datasets which have Datasources). I'm not sure if we can do a rename at this point, but we should strongly consider another module name that more clearly encompasses both Dashboards and Reports.

Reply:
We could do a another meta-module at some point to combine Data and Reports modules. Right now its only 5 cmdlets for dashboards, that might be a little small for its own module. That might grow when we add in PBIPS cmdlets.